src/util/mod.rs

```diff
@@ -34,3 +33,9 @@ pub mod address {
 
 #[deprecated(since = "0.30.0", note = "Please use crate::Error")]
 pub use crate::error::Error;
+
+/// The psbt module can now be accessed from the crate root.
+#[deprecated(since = "0.30.0", note = "Please use crate::psbt")]
+pub mod psbt {
+    pub use crate::psbt::*;
+}
```

Comment:
Is this here insteadd of `#[dperecated(...)] pub use crate::psbt` because the attribute doesn't work or something? I vaguely remember something like that but not sure.

Reply:
I did not actually think of doing it the way you suggest but I checked it just now the attribute does not function correctly for some reason if done that way.